Refresh and improve the tracking documentation (#239)
opengisch/QField-docs: documentation/how-to/tracking.en.md

# Tracking

QField allows you to create points, lines, and polygons from tracking your position.

## Activate tracking

:material-tablet-android:{ .device-icon } Fieldwork

![type:video](https://player.vimeo.com/video/499958163)

To activate tracking, you must first make sure the positioning is active. Then,
open the side dashboard, long-press on a legend layer within which you want to
save your tracks to, and select the *Setup tracking* button to configure
the tracking session.

!![](../assets/images/tracking-layer-properties.png)

Two constraints are available to limit the number of vertices recorded

> during tracing:

Reviewer: ```suggestion
during tracking:
```

> - A minimum time interval; and

Reviewer: ```suggestion
- A minimum time interval
```

- A minimum distance.

!![](../assets/images/tracking-settings.png)

> Once you have finished configuring your tracing session, hit the *Start tracking*

Reviewer: ```suggestion
Once you have finished configuring your tracking session, hit the *Start tracking*
```

button to begin recording. At this stage, a feature form will appear, which allows
you to define the attributes of the feature(s) that will be created while QField
tracks your position.

For line and polygon layers, a single feature - its geometry formed of vertices
from the recorded positions - will be create per tracking session.  For point
layers, a feature will be created for each recorded position, with attribute
values entered in the form remembered across features.

!!! note

>     QField will skip the feature form step if the layer is configured to hide all attributes. This can streamline the process of starting tracking sessions.

Reviewer: ... or if the form has been configured to be hidden in the attribute form settings

![image](https://user-images.githubusercontent.com/588407/187025845-bd2d86c3-ea46-4e12-951e-3302207d3572.png)

Author: @m-kuhn , we'll have to make this true by the time we publish 2.3 :)

Reviewer: It's already respected for simple feature creation IIRC

Author: @m-kuhn , it is now a truth: https://github.com/opengisch/QField/pull/3247/commits/e1e98bb94bb034f48f6850b32865c785485ac9a4 :)

## Tracking underway

Once a tracking session has been setup, a badge will appear in the side dashboard
legend next to the layer(s) against which tracking is being recorded.

!![](../assets/images/tracking-badge.png)

> The feature(s) created by the tracking session is/are saved on every vertex recorded.

Reviewer: ```suggestion
The features created by the tracking session are saved on every vertex recorded.
```
Easier to read and correct too
